### ISSUE TITLE
Reorganize the project root folder

### ISSUE THREAD

**Issue by lorenzonodari** [Koldar/crashc#74]
In our root folders we have a lot of files, maybe we should add some folders in order to better organize them, something like "scripts" or "info". What do you think?

**Comment by Koldar** [comment]
I agree with you. Actually our problem is with the building system as well: some commands have to be executed the root folder (i.e. ./testEverything.bash) while the other needs to be executed in "build/Debug". We indeed need to reorganize the folder, but this shouldn't be done in crashc 1.1 (at least not rght now). Feel free to think about a solution for the next release! :)
